contributing/PROCESS.md

```diff
@@ -106,12 +106,12 @@ In addition, contributors are required to complete either the Individual
 or Corporate Contribution License Agreement. Please contact one of the
 trusted committers for more information.
 
-## Templates and Guidelines
+## Futher reading
 
-- Coding standards [CODING_STANDARDS.md](CODING_STANDARDS.md)
-- Pull Requests: [PULL_REQUESTS.md](PULL_REQUESTS.md)
-- Commits/Commit messages: [COMMITS.md](COMMITS.md)
-- Code Reviews: [CODE_REVIEWS.md](CODE_REVIEWS.md)
+- [Coding standards](CODING_STANDARDS.md)
+- [Pull Requests](PULL_REQUESTS.md)
+- [Commits and commit messages] [COMMITS.md](COMMITS.md)
```

Comment:
[fix]
```suggestion
- [Commits and commit messages](COMMITS.md)
```

Reply:
I would swear on my life I tested this! Good spot, thanks!